app/src/main/java/com/tip/lunchbox/view/fragment/SearchFragment.java

```diff
@@ -15,7 +15,7 @@
 import com.tip.lunchbox.databinding.FragmentSearchBinding;
 import com.tip.lunchbox.view.adapter.CollectionsAdapter;
 import com.tip.lunchbox.view.adapter.RestaurantAdapter;
-import com.tip.lunchbox.viewmodel.SearchFramentViewModel;
+import com.tip.lunchbox.viewmodel.SearchFragmentViewModel;
 
```

Comment:
@devansh-299 can you check if I have covered all of the errors I will rebase if this is all